src/init/features/apphosting/index.ts

```diff
@@ -27,8 +27,12 @@ import { DEFAULT_REGION } from "./constants";
 import { ensure } from "../../../ensureApiEnabled";
 import * as deploymentTool from "../../../deploymentTool";
 import { DeepOmit } from "../../../metaprogramming";
+import { AppPlatform, createWebApp, listFirebaseApps } from "../../../management/apps";
+import * as inquirer from "inquirer";
+import * as fuzzy from "fuzzy";
 
 const DEFAULT_COMPUTE_SERVICE_ACCOUNT_NAME = "firebase-app-hosting-compute";
+const CREATE_NEW_WEB_APP_CHOICE = "CREATE_NEW_WEB_APP";
```

Comment:
nit: can we call this "CREATE_NEW_FIREBASE_WEB_APP"? Web app is an overloaded term and there was a time when we referred to app hosting backends as web apps (and sometimes we still do)

Reply:
ah good call out! Updated the naming here.